### PR TITLE
Fixing check_rpmspec_collection.sh

### PR DESCRIPTION
**Pull request by nhosoi**
Adjusting the expected behaviour based on the comment
https://src.fedoraproject.org/rpms/linux-system-roles/pull-request/4#comment-72412

### CODE REVIEW

File: check_rpmspec_collection.sh [hunk 1]
```diff
@@ -41,7 +42,7 @@ run() {
 }
 
 run "--without ansible" "without_ansible" 1
-run "--without ansible --with collection_artifact" "without_ansible-with_collection_artifact" 1
+run "--without ansible --with collection_artifact" "without_ansible-with_collection_artifact" 0
 run "--without ansible --without collection_artifact" "without_ansible-without_collection_artifact" 1
 run "--without collection_artifact" "without_collection_artifact" 1
 run "--with ansbile --without collection_artifact" "with_ansible-without_collection_artifact" 1
```
Comment (richm) [inline check_rpmspec_collection.sh:48]: There are a couple of places where you use `ansbile` instead of `ansible` that should be fixed

Comment (nhosoi) [reply check_rpmspec_collection.sh]: Thank you, @richm. There were plenty of them... :(

File: check_rpmspec_collection.sh [hunk 2]
```diff
@@ -41,11 +42,11 @@ run() {
...
 run "--without ansible" "without_ansible" 1
-run "--without ansible --with collection_artifact" "without_ansible-with_collection_artifact" 1
+run "--without ansible --with collection_artifact" "without_ansible-with_collection_artifact" 0
 run "--without ansible --without collection_artifact" "without_ansible-without_collection_artifact" 1
 run "--without collection_artifact" "without_collection_artifact" 1
-run "--with ansbile --without collection_artifact" "with_ansible-without_collection_artifact" 1
+run "--with ansible --without collection_artifact" "with_ansible-without_collection_artifact" 1
 run "" "no_opt" 1
-run "--with ansbile" "with_ansible" 1
+run "--with Ansible" "with_ansible" 1
```
Comment (richm) [inline check_rpmspec_collection.sh:50]: capital `A`?

Comment (nhosoi) [reply check_rpmspec_collection.sh]: *grrr...* Sorry, I mistakenly pressed a wrong number in `aspell check`... (;_;)

**Review by richm — APPROVED** [review]
lgtm

**Comment by nhosoi** [comment]
Thank you, @richm !

**Pull request merged**